plugins/mk/qml/Keyboard_mk_url_search.qml

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2013 Canonical Ltd.
```

Comment:
^

Reply:
to what tho?

Reply:
Since I was a bit too busy and don't even have the machine nearby, @Fuseteam did the work for me. :)